apps/web-giddh/src/app/verify-subscription-transfer-ownership/verify-subscription-transfer-ownership.component.html

```diff
@@ -40,17 +40,50 @@ <h2 class="modal-title">Subscription Transfer</h2>
                 </p>
             </div>
             <div class="modal-footer">
+                <button aria-label="yes" mat-stroked-button color="primary" (click)="onSubmit($event)">
+                    Accept Subscription
```

Comment:
translation

Reply:
We are using it in new signups where the translation module is not loaded.